src/org/opendatakit/briefcase/ui/reused/UI.java

```diff
@@ -62,24 +59,18 @@
 public class UI {
   private static final Font ic_receipt = FontUtils.getCustomFont("ic_receipt.ttf", 16f);
 
-  @SuppressWarnings("checkstyle:AvoidEscapedUnicodeCharacters")
-  public static JButton buildDetailButton(FormStatus form) {
+  public static DetailsStatusButton buildDetailButton(FormStatus form) {
```

Comment:
I would expect this to be a static factory in `DetailsStatusButton`

Reply:
Agreed. Made the change.